### PR TITLE
FUSEQE-1532

### PR DESCRIPTION
**Pull request by avano**
//skip-ci

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java [hunk 1]
```diff
@@ -195,4 +206,97 @@ private static void patchImageStreams() {
...
+                json.append(obj.substring(0, obj.length() - 1));
+                // Jackson can't serialize Map that is in the List that is in Map's Object value properly, therefore creating the json snippet manually here
+                spec.append("\"spec\":{").append(jsonKeyValue("import", "true")).append(",")
+                        .append("\"images\":[{\"from\":{").append(jsonKeyValue("kind", "DockerImage")).append(",").append(jsonKeyValue("name", imageStreamImport.getName()))
+                        .append("},").append("\"to\":{").append(jsonKeyValue("name", "1.3")).append("},\"importPolicy\":{").append(jsonKeyValue("insecure", "true")).append("},\"referencePolicy\":{")
+                        .append(jsonKeyValue("type", "")).append("}}]},\"status\":{}}");
+                json.append(",").append(spec.toString());
+                return json.toString();
+            } catch (JsonProcessingException e) {
+                e.printStackTrace();
```
Comment (mcada) [inline utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java]: maybe fail(...) here, it will probably never be called, but just in case?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**